.github/workflows/scheduled.yml

```diff
@@ -454,9 +454,9 @@ jobs:
   spark-aggregate-fuzzer-run:
     name: Spark Aggregate Fuzzer
     runs-on: ubuntu-latest
-    container: ghcr.io/facebookincubator/velox-dev:centos9
+    container: ghcr.io/facebookincubator/velox-dev:spark-server
     needs: compile
-    timeout-minutes: 60
+    timeout-minutes: 120
```

Comment:
Why is this change necessary? The usual duration is still 15/30mins. Did you actually see a timeout?

Reply:
Got it. This change is not necessary and just reverted it.